WordPressAuthenticator.podspec

```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressAuthenticator'
-  s.version       = '3.2.1'
+  s.version       = '3.2.2'
```

Comment:
The versioning here should follow our guidelines and issue a new beta. I'll share more details on slack, thank you!

Reply:
@Gio2018 Updated the podspec version to `3.2.2-beta.1` 9bee712d8117e28432c4c1e1eb1fbb89d6156b12